<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorCaseEditClient</class>
 <widget class="QWidget" name="doctorCaseEditClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>568</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>879</width>
    <height>568</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>病例编辑</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 主窗口样式 */
QWidget {
    font-family: &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;, sans-serif;
    background-color: #f8fafc;
}

/* 标签样式 */
QLabel {
    background-color: transparent;
    color: #374151;
    font-size: 16px;
    font-weight: 500;
    padding: 8px 0px;
}

/* 输入框样式 */
QLineEdit {
    background-color: white;
    border: 1px solid #e2e8f0;
    border-radius: 8px;
    padding: 12px;
    font-size: 14px;
    color: #1f2937;
    selection-background-color: #4a90e2;
    selection-color: white;
    min-height: 40px;
}

QLineEdit:focus {
    border: 1px solid #4a90e2;
    background-color: #f0f7ff;
    outline: none;
}

QLineEdit:hover {
    border: 1px solid #cbd5e1;
}

QLineEdit:read-only {
    background-color: #f1f5f9;
    color: #64748b;
    border: 1px solid #e2e8f0;
}

/* 按钮样式 */
QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: 500;
    transition: all 0.2s ease;
    min-width: 80px;
    min-height: 32px;
}

QPushButton:hover {
    background-color: #3a7bc8;
    transform: translateY(-1px);
}

QPushButton:pressed {
    background-color: #2c6bb6;
    transform: translateY(1px);
}

QPushButton:disabled {
    background-color: #cbd5e1;
    color: #94a3b8;
}

/* 编辑按钮特殊样式 */
#pushButton {
    background-color: white;
    color: #4a90e2;
    border: 1px solid #4a90e2;
}

#pushButton:hover {
    background-color: #4a90e2;
    color: white;
}

#pushButton:pressed {
    background-color: #3a7bc8;
    border-color: #3a7bc8;
}

/* 保存按钮特殊样式 */
#pushButton_2 {
    background-color: #10b981;
    color: white;
}

#pushButton_2:hover {
    background-color: #0d966c;
}

#pushButton_2:pressed {
    background-color: #0b845a;
}

/* 布局容器样式 */
#verticalLayoutWidget {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e2e8f0;
    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.05);
}

#verticalLayoutWidget_2 {
    background-color: transparent;
}

/* 布局样式 */
QVBoxLayout {
    spacing: 15px;
}
   </string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>541</width>
     <height>372</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="mainLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>66</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入主诉内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nowLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>66</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入现病史内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pastLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>66</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入既病史内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="checkLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>66</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入体格检查结果</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="diagnoseLineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>66</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入诊断结果</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>151</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>主诉</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>现病史</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>既病史</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>体格检查</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>-1</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="text">
       <string>诊断</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>122</width>
     <height>54</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>120</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
